blocks/rss-alexa-feature-block/features/rss/xml.js

```diff
@@ -72,7 +72,10 @@ const rssTemplate = (
           guid: s._id,
           pubDate: s[pubDate],
           ...(s.content_elements &&  {
-            enclosure: jmespath.search(s, 'content_elements[?type==audio].streams[].url[0]') || '',
+            enclosure: {
+              '@href': `jmespath.search(s, 'content_elements[?type==audio].streams[].url[0]') || ''`,
```

Comment:
this can possibly return `undefined`, need to check if there is a an audio type element first.